.drone.yml

```diff
@@ -1297,88 +1283,3 @@ trigger:
 depends_on:
 - default-linux-amd64
 - default-windows-ltsc2022
----
-kind: pipeline
-name: publish
-
-platform:
-  os: linux
-  arch: amd64
-
-steps:
-- name: chart-promote
-  image: rancher/dapper:v0.6.0
-  commands:
-  - dapper chart/copy
-  volumes:
-  - name: docker
-    path: /var/run/docker.sock
-  when:
-    event:
-    - promote
-    target:
-    - promote-stable
-
-- name: chart-publish
-  image: plugins/gcs
-  settings:
-    acl:
-    - allUsers:READER
-    cache_control: "public,no-cache,proxy-revalidate"
-    source: bin/chart
-    target: releases.rancher.com/server-charts
-    token:
-      from_secret: google_auth_key
-  when:
-    event:
-    - promote
-    target:
-    - promote-stable
-
-volumes:
-- name: docker
-  host:
-    path: /var/run/docker.sock
-
-trigger:
-  event:
-  - promote
----
-kind: pipeline
-name: promote-docker-image
```

Comment:
This pipeline is not related to charts, do we want to include it in this PR?  Or if it is requested, I suggest to update the commit message to make it clear.

Reply:
OK, this is requested. :)

Reply:
Discussed in slack, it was agreed to remove all steps related to promoting charts and images, as they will likely fail